test/integration/controllermanager/project/project/project_suite_test.go

```diff
@@ -133,6 +138,22 @@ var _ = BeforeSuite(func() {
 		RateLimiter: workqueue.NewWithMaxWaitRateLimiter(workqueue.DefaultControllerRateLimiter(), 100*time.Millisecond),
 	}).AddToManager(mgr)).To(Succeed())
 
+	By("Register CloudProfile controller")
+	Expect((&cloudprofile.Reconciler{
+		Config: config.CloudProfileControllerConfiguration{
+			ConcurrentSyncs: ptr.To(5),
+		},
+	}).AddToManager(mgr)).To(Succeed())
+
+	By("Register NamespacedCloudProfile controller")
+	Expect((&namespacedcloudprofile.Reconciler{
+		Config: config.NamespacedCloudProfileControllerConfiguration{
+			ConcurrentSyncs: ptr.To(5),
+		},
+	}).AddToManager(ctx, mgr)).To(Succeed())
```

Comment:
Maybe you can add a comment why these controllers are added to the integration test of the `Project` controller. I think it's not immediately obvious.